### ISSUE TITLE
In README, Change "PolyMC" to "Prism Launcher"

### ISSUE THREAD

**Issue by SylvKT** [FeralInteractive/gamemode#415]
**Is your feature request related to a problem? Please describe.**
The head of PolyMC has begun "removing wokeness" and slowly started allowing bigotry in the community, leading to a new launcher called "[Prism Launcher](https://prismlauncher.org/)" being created.

**Describe the solution you'd like**
Under "Apps with GameMode integration → Others", replace the reference to PolyMC with Prism Launcher.

**Comment by barfin** [comment]
I totally agree with this change, PolyMC is a dead project and shouldn't be referenced anymore

**Comment by afayaz-feral** [comment]
Fixed in 1f28880
